tests/datavolume_test.go

```diff
@@ -119,6 +119,54 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 		)
 	})
 
+	Describe("Verify DataVolume with block mode", func() {
```

Comment:
I am not seeing you create a block mode PV to bind to in these DV tests, and thus they are failing.

Reply:
added PV

Reply:
I would really do this in its own Describe, so you can put the create/remove of the PV in a BeforeEach and AfterEach. Because if the test fails in this scenario it will never remove the PV.